pkg/agent/iptables/iptables.go

```diff
@@ -42,24 +56,35 @@ var (
 	// Antrea uses 10th for SNAT.
 	masqueradeBit   = uint(10)
 	masqueradeValue = 1 << masqueradeBit
-	masqueradeMark  = fmt.Sprintf("%#08x/%#08x", masqueradeValue, masqueradeValue)
+	masqueradeMark  = fmt.Sprintf("%#x/%#x", masqueradeValue, masqueradeValue)
+
+	// RtTblSelectorValue selects which route table to use to forward service traffic back to host gateway gw0.
+	RtTblSelectorValue = 1 << 11
+	rtTblSelectorMark  = fmt.Sprintf("%#x/%#x", RtTblSelectorValue, RtTblSelectorValue)
 )
 
 // Client knows how to set up host iptables rules Antrea requires.
 type Client struct {
 	ipt         *iptables.IPTables
 	hostGateway string
+	nodeConfig  *types.NodeConfig
+	serviceCIDR *net.IPNet
+	encapMode   config.TrafficEncapModeType
+	store       *sync.Map
 }
 
+// key is chain, value is rules in the chain
+type chainRules map[string][][]string
```

Comment:
Could this comment explain why the value is `[][]string` and not `[]string`?

Reply:
done